src/Sfa.Das.Sas.Indexer.ApplicationServices/Shared/Settings/AppServiceSettings.cs

```diff
@@ -64,6 +66,13 @@ public AppServiceSettings(IProvideSettings settingsProvider)
 
         private string VstsGitBaseUrl => _settings.GetSetting("VstsGitBaseUrl");
 
+        public List<string> FrameworksExpiredRequired => GetFrameworksList(_settings.GetSetting("FrameworksExpiredRequired"));
+
+        private List<string> GetFrameworksList(string frameworkIdList)
```

Comment:
If frameworkIdList is null, it'll throw a NullReferenceException. Can the _settings return a null?

Reply:
Fixed, now if frameworkIdList is null, we return an empty list